services/core-web/src/components/common/DocumentTable.tsx

```diff
@@ -127,11 +126,11 @@ export const DocumentTable = ({
   const documents = parseDocuments(props.documents ?? []);
 
   useEffect(() => {
-    const isMultiSelect = documents.every((doc) =>
-      doc.allowed_actions.find((a) => a === "MultiSelect")
+    const isBulkArchive = documents.every((doc) =>
+      doc.allowed_actions.find((a) => a === "Archive file")
```

Comment:
nitpick: use the constant for "Archive file" rather than the string itself

Reply:
This has been changed to use `FileOperations.Archive` now.